scripts/Editor/MCpiroman.UnityNativeTool.Editor.asmdef

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "MCpiroman.UnityNativeTool.Editor",
```

Comment:
don't know if it matters, but my nick is not spelled in the same casing (if it does I prefer lowercase)

Reply:
I thought I saw it written somewhere like that, I'll change it to all lowercase.